modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -70,3 +70,30 @@
 	else
 		to_chat(M, span_notice("[pick("I feel oddly calm.", "I feel relaxed.", "Mew?")]"))
 	..()
+
+#define DERMAGEN_SCAR_FIX_AMOUNT 10
+
+/datum/reagent/medicine/dermagen
+	name = "Dermagen"
+	description = "Heals scars formed by past physical trauma when applied. Minimum 10u needed, only works when applied topically."
+	reagent_state = LIQUID
+	color = "#FFEBEB"
+	ph = 6
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/medicine/dermagen/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume, show_message = TRUE)
+	. = ..()
+	if(!iscarbon(exposed_mob))
+		return
+	var/mob/living/carbon/scarred = exposed_mob
+	if(scarred.stat == DEAD)
+		show_message = 0
+	if(!(methods & (PATCH|TOUCH|VAPOR)))
+		return
```

Comment:
Since I was lazy as hell and this was a synthflesh copy/paste, synthflesh is the only spot in the codebase that still uses `show_message = 0` instead of `FALSE`. Kind of amusing.